# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/rene/guile/guile-missing-dotlibs-dir.patch
# ROCK Linux is Copyright (C) 1998 - 2005 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./pre-inst-guile.in.orig	2004-11-09 17:16:44.000000000 +0100
+++ ./pre-inst-guile.in	2004-12-16 14:15:17.347930016 +0100
@@ -36,7 +36,7 @@
 # Code:
 
 # config
-subdirs_with_ltlibs="libguile-ltdl srfi guile-readline"       # maintain me
+subdirs_with_ltlibs="libguile-ltdl libguile-ltdl/.libs srfi guile-readline"       # maintain me
 
 # env (set by configure)
 top_srcdir="@top_srcdir_absolute@"
@@ -74,6 +74,7 @@
 done
 LTDL_LIBRARY_PATH="${ltdl_prefix}$LTDL_LIBRARY_PATH"
 export LTDL_LIBRARY_PATH
+export LD_LIBRARY_PATH="$LTDL_LIBRARY_PATH:$LD_LIBRARY_PATH"
 
 # set GUILE (clobber)
 GUILE=${top_builddir}/libguile/guile
